### ISSUE TITLE
Builder can't build water water blocks

### ISSUE THREAD

**Issue by daxxog** [cabaletta/baritone#477]
The builder can't seem to place water blocks when I run a schematic with water source blocks in it. It fails with "unable to do it". I have two water buckets in my hotbar. I made a [schematic](https://github.com/cabaletta/baritone/files/3365227/water.schematic.zip) with two water source blocks side by side to illustrate the issue. Ideally baritone would be smart enough to do water duplication and make unlimited water from two buckets. I haven't done Java in years, looking to brush up on my skills. 
If you would like some help with this issue I might work on it and submit a PR. Been reading your code in my spare time, very interested in this project.

**Comment by l1ving** [comment]
No, baritone doesn't have support for this functionality

**Comment by daxxog** [comment]
But it would be a nice feature for automated underground farm building.

**Comment by sktedro** [comment]
Yep that would really be great. Placing often water takes a "long" time when building. I could even imagine a method that, in case no water bucket is in the inventory, approaches a nearby infinite water source and refills it.
